h/static/scripts/karma.config.js

```diff
@@ -57,10 +59,16 @@ module.exports = function(config) {
       extensions: ['.coffee'],
       noParse: [require.resolve('./vendor/katex')],
       configure: function (bundle) {
-        bundle
-          .transform('coffeeify')
-          .plugin('proxyquire-universal');
+        bundle.plugin('proxyquire-universal');
       },
+
+      transform: [
+        'coffeeify',
+        istanbul({
+          ignore: ['**/node_modules/**', '**/katex.js', '**/*.html', '**/*.svg'],
+          instrumenter: require('isparta'),
```

Comment:
Can you add a comment here to explain why isparta is being used instead of the original instrumenter?

The [README](https://github.com/douglasduteil/isparta) describes it as a "code coverage tool for ES6 using Babel", but the problem we're trying to solve here is different; mapping lines in transpiled CoffeeScript code back to original locations in the JS source code.

AAUI, this does actually work because isparta [transforms the function/statement/branch location maps](https://github.com/douglasduteil/isparta/blob/master/src/instrumenter.js) by reading the source maps in the generated code, so even though it was designed for use with Babel it actually works with any transpiled code that includes sourcemaps.

Perhaps in future we might want to consider a plugin that is designed more specifically for this purpose, eg. [remap-istanbul](https://github.com/SitePen/remap-istanbul)?